inc/form.class.php

```diff
@@ -2001,6 +2001,7 @@ public function showForCentral() {
       }
 
       $form_query = PluginFormcreatorForm::getFormListQuery();
+      $form_query['DISTINCT'] = true;
```

Comment:
Maybe this change should be in PluginFormcreatorForm::getFormListQuery() ?

Reply:
Indeed, I think it should be, in order to be a more global solution.